src/MessageEditor.js

```diff
@@ -37,6 +39,18 @@ class MessageEditor extends React.Component {
     }
   }
 
+  updateMessages = () => {
+    this.setState({
+      messages: [],
+      gauges: []
+    })
```

Comment:
Wait, did we not restructure the code so that this no longer was needed?  I am confused.

Reply:
Still seems to be needed. Still not sure why :/